include/PandoraMonitoring.h

```diff
@@ -222,13 +222,17 @@ class PandoraMonitoring
 
     /**
      *  @brief  Show the Eve Event-display and pause.
+     *
+     *  @param  shouldInitializeGui Does the GUI need to be initialized. False if initialized elsewhere.
      */
-    void ViewEvent(const bool needsInitialsation = true);
+    void ViewEvent(const bool shouldInitializeGui = true);
```

Comment:
Hi Ryan, This is nice functionality to have, good work.  One thing I would say is that I think you may be able to remove the shouldInitializeGui bool.  From past experiences, I remember the InitializeEve gets called several times (once per each type of object being drawn) and it already has a check within it to see if the event has been initialised (and if it has to not bother redrawing it).  So it might be worth testing without this bool to see if things still work fine for you.  Cheers, Steve

Reply:
I'll do a more thorough test overnight (diff a folder of images vs some I did a few days ago), but looks like its not needed as between `pEveManager` not being null and `m_openEveEvent` being set (and only being unset at the end of `ViewEvent`), running for the second time doesn't do anything, and the cameras don't move during saving out images.

Thanks for the tip!

Reply:
No worries!  I found it a bit odd the way the TEve manager worked in the past and thought I'd pass on the limited wisdom I gained!